api/util/common-test-fixtures.ts

```diff
@@ -0,0 +1,124 @@
+import { SQSEvent } from "aws-lambda";
+import { Network } from "../../models/cloud-service-providers";
+import { CostRequest } from "../../models/cost-jobs";
+import { ProvisionRequestType } from "../../models/provisioning-jobs";
+import * as crypto from "crypto";
+
+// provisioning fixtures
+export const fundingSources = [
+  {
+    taskOrderNumber: "1234567890123",
+    clin: "9999",
+    popStartDate: "2021-07-01",
+    popEndDate: "2022-07-01",
+  },
+];
+
+export const operators = ["admin1@mail.mil", "superAdmin@mail.mil"];
+
+export const provisioningBodyNoPayload = {
+  jobId: "81b31a89-e3e5-46ee-acfe-75436bd14577",
+  userId: "21d18790-bf3e-4529-a361-460ee6d16e0b",
+  portfolioId: "b02e77d1-234d-4e3d-bc85-b57ca5a93952",
+  operationType: ProvisionRequestType.ADD_OPERATORS,
+  targetCsp: {
+    name: "CSP_A",
+    uri: "http://www.somecspvendor.com/api/atat",
```

Comment:
Can we perhaps change this to 
```suggestion
    uri: "https://cspa.example.com/api/atat",
```

Or something like that? Having HTTPS clears up a Sonar finding and I'd feel better if we didn't put potentially-valid URLs in our code base (someone might buy `somecspvendor.com` some day but `example.com` is reserved).